src/i18n/resources/en.js

```diff
@@ -175,6 +175,7 @@ The FRA team fra@fao.org
     forest: 'Forest',
     otherWoodedLand: 'Other wooded land',
     otherLand: 'Other land',
+    remainingLand: 'Remaining land',
```

Comment:
Don't we have this already ?

Reply:
![image](https://user-images.githubusercontent.com/5508251/215708169-c2d8a7d3-0e9d-4e92-98f5-be603c2d0408.png)